tests/app/celery/test_scheduled_tasks.py

```diff
@@ -1146,3 +1154,44 @@ def test_successful_run(self, mocker, mock_prod_notify_api, notify_db_session, m
         ]
         csv_file = mock_zendesk_update_ticket.call_args_list[0][1]["comment"].attachments[0].filedata
         assert csv_file.read() == 'result,text,comma\r\n1,something else,"quote,text"\r\n'
+
+
+# need to make sure this isnt prefixed with "Test" or pytest will collect and run the base class
+class DvlaScheduledTaskTests:
+    dvla_method_import_path = None
+    scheduled_task_function = None
```

Comment:
Do we have this pattern anywhere else? It's fine although I suspect parameterising the tests is probably easier to read/grok and more consistent? And we don't have to comment about the specific prefix avoidance thing